### PR TITLE
support to display the relevant database info when switching databases

### PR DESCRIPTION
**Pull request by obiscr**
Support switching databases to display the relevant database configurations during the initial installation

When switching to Mysql, use:
Username: root
Password: root
Database host: db:3306
image

When switching to Postgres, use
Username: postgres
Password: postgres
Database host: db:5432
image

### CODE REVIEW

File: ui/src/pages/Install/index.tsx [hunk 1]
```diff
@@ -131,11 +131,35 @@ const Index: FC = () => {
     },
   });
 
+  const updateFormData = (params: FormDataType) => {
+    if (step === 2) {
```
Comment (shuashuai) [inline ui/src/pages/Install/index.tsx:135]: Monitoring changes in value of db_type is more accurate

 if (params.db_type?.value !== formData.db_type.value) {
  ...
}

File: ui/src/pages/Install/index.tsx [hunk 2]
```diff
@@ -131,11 +131,35 @@ const Index: FC = () => {
...
+      let updatedFormData = formData;
+      if (params.db_type.value === 'mysql') {
```
Comment (shuashuai) [inline ui/src/pages/Install/index.tsx:137]: params.db_type?.value

File: ui/src/pages/Install/index.tsx [hunk 3]
```diff
@@ -131,11 +131,35 @@ const Index: FC = () => {
...
+        updatedFormData = {
+          ...updatedFormData,
+          db_username: { ...updatedFormData.db_username, value: 'root' },
+          db_password: { ...updatedFormData.db_password, value: 'root' },
+          db_host: { ...updatedFormData.db_host, value: 'db:3306' },
+        };
+      } else if (params.db_type.value === 'postgres') {
```
Comment (shuashuai) [inline ui/src/pages/Install/index.tsx:144]: params.db_type.value

File: ui/src/pages/Install/index.tsx (hunk 1 above)
Comment (obiscr) [reply ui/src/pages/Install/index.tsx]: Thanks, It make sense. The latest changed has been updated at https://github.com/apache/incubator-answer/pull/1016/commits/b18016fd97f3112860dd4beb5ae08a177b62b128

**Review by shuashuai — APPROVED** [review]
Thanks for the contribution

**Pull request merged**